packages/headless/src/features/attached-results/attached-results-state.ts

```diff
@@ -0,0 +1,59 @@
+export interface AttachedResult {
+  /**
+   * Specifies the language of an attached Salesforce Knowledge Article.
+   */
+  articleLanguage?: string;
+  /**
+   * Specifies the version number of an attached Salesforce Knowledge Article.
+   */
+  articleVersionNumber?: string;
+  /**
+   * Specifies the publish status of an attached Salesforce Knowledge Article.
+   */
+  articlePublishStatus?: string;
+  /**
+   * The Id of the record (Case) the result is attached to.
+   */
+  caseId: string;
+  /**
+   * The Id of an attached Salesforce Knowledge Article.
+   */
+  knowledgeArticleId?: string;
+  /**
+   * The permanentId, the identifier of an attached result.
+   */
+  permanentId?: string;
+  /**
+   * The url of the attached result.
+   */
+  resultUrl: string;
+  /**
+   * The source name where the attached result comes from.
+   */
+  source: string;
```

Comment:
Is this for the Coveo source? I'm wondering why it's required while most of the others aren't.

Reply:
yea I just removed it actually it was breaking some other thing to be required :D